QPM/Providers/RemoteQPMDependencyResolver.cs

```diff
@@ -26,6 +28,7 @@ internal class RemoteQPMDependencyResolver : IDependencyResolver
         private readonly QPMApi api;
         private readonly Dictionary<RestoredDependencyPair, SharedConfig> cached = new Dictionary<RestoredDependencyPair, SharedConfig>();
         private readonly AndroidMkProvider androidMkProvider;
+        private static readonly Linker Linker = new();
```

Comment:
I personally like having private variables (including static ones) be lowercase, uppercase reserved for properties, methods, and types. Would appreciate this being changed, though this is all nitpicks now.

Reply:
I personally prefer it too but I just rode along with Rider's .NET variable practices (get it? xd)